docs/en/index.rst

```diff
@@ -3,7 +3,12 @@
 Phinx Documentation
 ===================
 
-Phinx makes it ridiculously easy to manage the database migrations for your PHP app. In less than 5 minutes, you can install Phinx using Composer and create your first database migration. Phinx is just about migrations without all the bloat of a database ORM system or application framework.
+Phinx makes it ridiculously easy to manage the database migrations for your PHP
+app. In less than 5 minutes, you can install Phinx using Composer and create
+your first database migration. Phinx is just about migrations without all the
+bloat of a database ORM system or application framework.
+
+Phinx requires a 64-bit version of PHP to function.
```

Comment:
The only real change here is the addition of this line. I only added line breaks to the above paragraph.